.github/workflows/preview-release.yml

```diff
@@ -0,0 +1,132 @@
+            echo Preview build including new features waiting for release.  >>preview_changes.md
+            git show -s --date=format:^"Date: ```%%Y-%%m-%%d```  ^" --format=%%cd>>preview_changes.md
+            git show -s --format=^"Hash: ```%%h```  ^">>preview_changes.md
+            echo [**Download**]^(${{github.server_url}}/${{github.repository}}/releases/download/preview/SA.CLEO-v${{steps.custom_build.outputs.version}}+.zip^)  >>preview_changes.md
+            echo;>>preview_changes.md
+            echo ^<details^>>>preview_changes.md
+            echo   ^<summary^>Changes since %tag%^</summary^>>>preview_changes.md
+            echo;>>preview_changes.md
+            git show -s --format="  * %%s" %tag%..HEAD>>preview_changes.md
+            echo   [View commits]^(${{ github.server_url }}/${{ github.repository }}/compare/%tag%...preview^)>>preview_changes.md
+            echo;>>preview_changes.md
+            echo ^</details^>>>preview_changes.md
+          )
+          
+      - uses: actions/setup-node@v4
+        if: ${{ hashFiles('preview_changes.md') != '' }}
+        with:
+          node-version: lts/*
+        
+      - name: Configure msbuild
+        uses: microsoft/setup-msbuild@v2
+        
+      - name: Build CLEO Core
+        if: ${{ hashFiles('preview_changes.md') != '' }}
+        shell: cmd
+        run: |
+          set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
+          msbuild -m CLEO5.sln /property:Configuration=Release /property:Platform=GTASA
+          
+      - name: Build CLEO Plugins
+        if: ${{ hashFiles('preview_changes.md') != '' }}
+        shell: cmd
+        run: |
+          set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
+          msbuild -m cleo_plugins/CLEO_Plugins.sln /property:Configuration=Release /property:Platform=x86
+          
+      - name: Gather output files
+        if: ${{ hashFiles('preview_changes.md') != '' }}
+        id: prepare_archive
+        shell: cmd
+        run: |
+          @REM create output directory
+          mkdir .output\Release\cleo
+          mkdir .output\Release\cleo\.config
+          mkdir .output\Release\cleo\cleo_plugins
+
+          @REM copy files
+          copy preview_changes.md .output\Release\CLEO5_Preview.md
+          copy source\cleo_config.ini .output\Release\cleo\.cleo_config.ini
+          copy cleo_plugins\.output\*.cleo .output\Release\cleo\cleo_plugins
+          copy cleo_plugins\.output\*.ini .output\Release\cleo\cleo_plugins
+          copy cleo_plugins\Audio\bass\bass.dll .output\Release\bass.dll
+
+          @REM copy SDK
+          copy .output\Release\CLEO.lib cleo_sdk\CLEO.lib
+
+          @REM download Sanny Builder Library json
+          curl https://raw.githubusercontent.com/sannybuilder/library/master/sa/sa.json -o .output\Release\cleo\.config\sa.json
+          
+      - name: Pack archive
+        if: ${{ hashFiles('preview_changes.md') != '' }}
+        uses: thedoctor0/zip-release@0.7.6
```

Comment:
why this, and not what we use for releases?

Reply:
Interesting, they are the same now. I think the fork actually added support for Windows, or some other feature that was not available on the source.